.github/workflows/release-notes-presence-check.yml

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-name: Check PR Release Notes in Description
+name: Check PR Release Notes Presence Check
```

Comment:
```suggestion
name: Check PR Release Notes Presence in Description
```

Reply:
I am little torn on this. Adding the info into the name is good. But my experience is, that if it differs from filename or workflow name, it leads to confusion.

Reply:
ok, so let's change the job name, too. ok?

Reply:
It is the same now, isn't it?

Reply:
It is not. To be same there is need to rename the job name to "Release Notes Presence Check"

Reply:
Not it's not 🙄